pkg/gadgets/trace/network/types/types.go

```diff
@@ -24,31 +24,31 @@ import (
 type Event struct {
 	eventtypes.Event
 
-	PktType string `json:"pkt_type,omitempty"`
+	PktType string `json:"pktType,omitempty"`
 	Proto   string `json:"proto,omitempty"`
 	IP      string `json:"ip,omitempty"`
 	Port    int    `json:"port,omitempty"`
 
 	/* pod, svc or other */
-	RemoteKind string `json:"remote_kind,omitempty"`
+	RemoteKind string `json:"remoteKind,omitempty"`
 
-	PodHostIP string            `json:"pod_host_ip,omitempty"`
-	PodIP     string            `json:"pod_ip,omitempty"`
-	PodOwner  string            `json:"pod_owner,omitempty"`
-	PodLabels map[string]string `json:"pod_labels,omitempty"`
+	PodHostIP string            `json:"podHostIp,omitempty"`
+	PodIP     string            `json:"podIp,omitempty"`
```

Comment:
`Ip` -> `IP`. (as done in Kubernetes https://github.com/kubernetes/api/blob/v0.24.3/core/v1/types.go#L3839).

Reply:
I see that in Kubernetes it's called with the full name always: `Service`, maybe we should switch to that.

Reply:
I was taking about it. Kubernetes always uses `Service` instead of the abbreviation, so if we want to be fully aligned we should name those like `remoteServiceNamespace` and so on.